scripts/exregional_get_mrms_files.sh

```diff
@@ -22,97 +24,106 @@ if [[ ! -d "$mrms_proc" ]]; then
 fi
 
 # Initialization
-start_init=${CDATE}${hh}
-#start_init=2020063000
+start_valid=${CDATE}${hh}
 
 # Forecast length
 fcst_length=${fhr_last}
-#fcst_length=36
 
-s_yyyy=`echo ${start_init} | cut -c1-4`  # year (YYYY) of initialization time
-s_mm=`echo ${start_init} | cut -c5-6`    # month (MM) of initialization time
-s_dd=`echo ${start_init} | cut -c7-8`    # day (DD) of initialization time
-s_hh=`echo ${start_init} | cut -c9-10`   # hour (HH) of initialization time
-start_init_ut=`date -ud ''${s_yyyy}-${s_mm}-${s_dd}' UTC '${s_hh}':00:00' +%s` # convert initialization time to universal time
-#echo "start_init_ut=${start_init_ut}"
+s_yyyy=`echo ${start_valid} | cut -c1-4`  # year (YYYY) of start time
+s_mm=`echo ${start_valid} | cut -c5-6`    # month (MM) of start time
+s_dd=`echo ${start_valid} | cut -c7-8`    # day (DD) of start time
+s_hh=`echo ${start_valid} | cut -c9-10`   # hour (HH) of start time
+start_valid_ut=`date -ud ''${s_yyyy}-${s_mm}-${s_dd}' UTC '${s_hh}':00:00' +%s` # convert start time to universal time
 
 end_fcst_sec=`expr ${fcst_length} \* 3600` # convert last forecast lead hour to seconds
-end_init_ut=`expr ${start_init_ut} + ${end_fcst_sec}` # calculate current forecast time in universal time
-#echo "end_init_ut=${end_init_ut}"
-#end_init=`date -ud '1970-01-01 UTC '${end_init_ut}' seconds' +%Y%m%d%H` # convert universal time to standard time
-#echo "end_init=${end_init}"
+end_valid_ut=`expr ${start_valid_ut} + ${end_fcst_sec}` # calculate current forecast time in universal time
 
-cur_ut=${start_init_ut}
+cur_ut=${start_valid_ut}
 current_fcst=0
 fcst_sec=`expr ${current_fcst} \* 3600` # convert forecast lead hour to seconds
 
-while [[ ${cur_ut} -le ${end_init_ut} ]]; do
-  cur_init=`date -ud '1970-01-01 UTC '${cur_ut}' seconds' +%Y%m%d%H` # convert universal time to standard time
-  echo "cur_init=${cur_init}"
+while [[ ${cur_ut} -le ${end_valid_ut} ]]; do
```

Comment:
This is all still good with the python script and doesn't pull too much each time?

Reply:
This change doesn't affect the original behavior (i.e., it pulls the same data as the original script).